services/integrationtest.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
```

Comment:
[nit-pick] can we mv this to `integration.sh`?  more consistent with `integration.yaml`.

Reply:
sure, done.